airbyte-integrations/bases/debezium/src/main/java/io/airbyte/integrations/debezium/internals/DebeziumPropertiesManager.java

```diff
@@ -53,6 +53,8 @@ protected Properties getDebeziumProperties() {
     // default values from debezium CommonConnectorConfig
     props.setProperty("max.batch.size", "2048");
     props.setProperty("max.queue.size", "8192");
+    props.setProperty("slave_net_timeout", "120");
```

Comment:
This is specific to MySQL so it should not be in the DebeziumPropertiesManager. It should be in `MySqlCdcProperties` class